Controllers/Stanley/Stanley_demo.py

```diff
@@ -35,6 +36,13 @@ def update_state(self, a, delta_f):
   def get_state(self):
     return self.x, self.y, self.psi, self.v
 
+  def get_front_state(self):
+      front_x = self.x + self.L * math.cos(self.psi)
+      front_y = self.y + self.L * math.sin(self.psi)
+      front_v = self.v / math.cos(self.delta_f)
```

Comment:
nit: `self.delta_f` seems should be updated by `self.delta_f = delta_f`.

Reply:
updated